test/tests/functional/pbs_fairshare.py

```diff
@@ -222,7 +226,7 @@ def test_fairshare_formula5(self):
         """
 
         self.set_up_resource_group()
-        a = {'log_filter': 2048, 'fair_share': "True All"}
+        a = {'log_filter': 2048, 'fair_share': "True ALL"}
```

Comment:
does this not require double quotes like #565?
or was it rejected because it was `All` instead of `ALL`

Reply:
it doesn't require double quotes. Yes the problem was "All" vs "ALL"
# fair_share
#       Schedule jobs based on usage and share values
#
#       PRIME OPTION
#

fair_share: false       ALL

Reply:
Thanks